__PROJECT NAME__/fastlane/CI/CIfile

```diff
@@ -63,7 +63,8 @@ platform :ios do
 
     crashlytics(
       ipa_path: "./build/#{ENV["PROJECT_NAME"]}.ipa",
-      groups: is_internal ? ["internal"] : ["braive-team"],
+      # note: - change the right statement to external group
+      groups: is_internal ? ["internal"] : [],
```

Comment:
hmm how about we add this to `README` and put in some alert logging in case we really miss this one 🤔